snapshot-labs/sx-ui
# feat: add veto support

Depends on: https://github.com/snapshot-labs/sx-subgraph/pull/30
Depends on: https://github.com/snapshot-labs/sx.js/pull/198
Closes: https://github.com/snapshot-labs/sx-ui/issues/684

## Test plan
- Set up dependencies (mana + subgraph)
- Create proposal with timelock execution where you are added as veto guardian.
- Vote on proposal, execute it.
- During timelock delay you should see option to veto it.
- Veto it.

## Screenshots

<img src="https://github.com/snapshot-labs/sx-ui/assets/1968722/896bac55-dce2-4859-9b49-7e66a9571fe5" width="600" />

<img src="https://github.com/snapshot-labs/sx-ui/assets/1968722/872e4cad-80a7-42b8-9598-fef952e39c0b" width="600" />
